bitmap-plasma/app/build.gradle

```diff
@@ -19,7 +19,7 @@ model {
 
     android.buildTypes {
         release {
-            isMinifyEnabled = false
+            minifyEnabled = false
```

Comment:
@rschiu it would be nice to update http://tools.android.com/tech-docs/new-build-system/gradle-experimental with a release + debuggeable example.